pkg/jaeger/store/trace_query.go

```diff
@@ -57,8 +60,16 @@ const (
 		_ps_trace.instrumentation_lib inst_lib ON s.instrumentation_lib_id = inst_lib.id
 	LEFT JOIN
 		_ps_trace.schema_url inst_lib_url ON inst_lib_url.id = inst_lib.schema_url_id
-	LEFT JOIN
-		_ps_trace.link lk ON lk.trace_id = s.trace_id AND lk.span_id = s.span_id %[4]s
+	LEFT JOIN LATERAL (
```

Comment:
is there a reason we don't use this same method for events? It seems to me that we should be using the same method for links and events if possible.